Makefile

```diff
@@ -1,28 +1,29 @@
+BUILDER_TAG?="noobaa-builder"
+NOOBAA_BASE_TAG?="noobaa-base"
+NOOBAA_TAG?="noobaa"
+TESTER_TAG?="noobaa-tester"
+POSTGRES_IMAGE?="centos/postgresql-12-centos7"
+MONGO_IMAGE?="centos/mongodb-36-centos7"
+
 CONTAINER_ENGINE?=$(shell docker version >/dev/null 2>&1 && echo docker)
 ifeq ($(CONTAINER_ENGINE),)
 	CONTAINER_ENGINE=$(shell podman version >/dev/null 2>&1 && echo podman)
 endif
+ifeq ($(CONTAINER_ENGINE),)
+	CONTAINER_ENGINE=$(shell lima nerdctl version >/dev/null 2>&1 && echo lima nerdctl)
+endif
 
 # see https://github.com/containerd/nerdctl/blob/main/docs/multi-platform.md
 # e.g use CONTAINER_PLATFORM=amd64 for building x86_64 on arm.
+CONTAINER_PLATFORM?="amd64"
```

Comment:
On my old Mac, the x86 one, `CONTAINER_PLATFORM?="amd64"` lead to:
```sh
$ make executable

## Build image noobaa-builder ...
docker build "--platform="amd64""  -f src/deploy/NVA_build/builder.Dockerfile   -t noobaa-builder .
[+] Building 0.8s (3/3) FINISHED
 => [internal] load .dockerignore                                                                                                                                                                                                      0.0s
 => => transferring context: 152B                                                                                                                                                                                                      0.0s
 => [internal] load build definition from builder.Dockerfile                                                                                                                                                                           0.0s
 => => transferring dockerfile: 2.18kB                                                                                                                                                                                                 0.0s
 => ERROR [internal] load metadata for quay.io/centos/centos:stream8                                                                                                                                                                   0.7s
------
 > [internal] load metadata for quay.io/centos/centos:stream8:
------
builder.Dockerfile:1
--------------------
   1 | >>> FROM quay.io/centos/centos:stream8
   2 |     LABEL maintainer="Liran Mauda (lmauda@redhat.com)"
   3 |
--------------------
ERROR: failed to solve: quay.io/centos/centos:stream8: no match for platform in manifest sha256:76dcf5724accefc182f93ef58c849a69fdbf937f9e94e7ab2230ab972d066d41: not found
make: *** [builder] Error 1
```
Commenting it out make it work. 
Do we want the default to be `"amd64"` ?

Reply:
Thanks for testing this, I really didn't know if this would also work out on real x86 arch.
Perhaps I should just check the current arch first and only set to x86 (amd64) if the current arch is arm...
WDYT?

Reply:
@liranmauda I tried to add a fix - can you try to run again?

Reply:
Sound Great.
Now it works for me
```sh
$ make executable

## Build image noobaa-builder ...
docker build   -f src/deploy/NVA_build/builder.Dockerfile   -t noobaa-builder .
[+] Building 0.3s (16/16) FINISHED
 => [internal] load .dockerignore                                                                                                                                                                                                      0.0s
 => => transferring context: 152B                                                                                                                                                                                                      0.0s
 => [internal] load build definition from builder.Dockerfile                                                                                                                                                                           0.1s
 => => transferring dockerfile: 2.18kB                                                                                                                                                                                                 0.0s
 => [internal] load metadata for quay.io/centos/centos:stream8                                                                                                                                                                         0.0s
 => [ 1/11] FROM quay.io/centos/centos:stream8
```